pages/_app.js

```diff
@@ -62,20 +80,31 @@ export default function App({ Component, pageProps }) {
     //If all Kana only search Reading
     if (wanakana.isKana(query)) {
       const results = wordList.filter((item) =>
-        searchedRegex.test(item.japanese[0].reading)
+        searchedRegex.test(item.japanese.reading)
       );
       setSearchResults(results);
     }
 
     //If Kanjis are included search Japanese Definition
     if (wanakana.isKanji(query) || wanakana.isMixed(query)) {
       const results = wordList.filter((item) =>
-        searchedRegex.test(item.japanese[0].word)
+        searchedRegex.test(item.japanese.word)
       );
       setSearchResults(results);
     }
   }
 
+  // Search Dictionary
+  useEffect(() => {
+    setDictionaryResults([]);
+
+    mutate();
```

Comment:
`mutate` is probably doing something useful here, but see if you can put it closer to the event that causes a change in your data. It's not ideal to have it here and listed as a dependency as well since the function itself never changes.

Reply:
I'll move it to the search bar :)